app/components/footer-nav.hbs

```diff
@@ -108,77 +108,52 @@
             <h5 class='footer-header' itemprop='name'>
               Stichting Sociëteit Flux
             </h5>
-            <table class='table table-sm'>
+            <table class='table table-sm mw-100'>
```

Comment:
By default, the `max-width` of table is `inherit`, which means it will take the `max-width` of the parent, on desktop being `33%` (width of `.col-md-4`). This causes the table to wrap text to new lines unnecessarily, most notably the address.